### DISCUSSION TITLE
copilot start error in vscode ( [error] Activating extension GitHub.copilot failed due to an error:)

### DISCUSSION THREAD

**Discussion by LimLLL** [community/community#30996] [Copilot Conversations] [answered]
tried to uninstall it multiple times, but it is ok when I installed the 1.38.6394 version, and this version can not be used currently since the update is required. Please help....
![0621951552c777973cec5ce7987891d](https://user-images.githubusercontent.com/59259075/187013684-c396a53b-1f6e-4b4d-bd16-3de4f980c748.jpg)
.

**Comment by davecheney** [comment, marked as answer]
@likeman66 i see you have upgraded to  copilot/1.43.6621 and are able to use Copilot. Is the issue resolved?

**Reply by LimLLL** [reply]
> @likeman66 i see you have upgraded to copilot/1.43.6621 and are able to use Copilot. Is the issue resolved?

thanks for the reply. yeah, it is working now after I deleted the old account auth file that was generated by older version of copilot

**Reply by YLSMW** [reply]
Hello, @likeman66 
I got the same issue. 
Could you please teach me the detailed solution of this?
What is the directory of the old account auth file and what's the prcedures of updating it?
Thank you in advance so much!

**Reply by davecheney** [reply]
@YLSMW please try updating to the latest vscode and copilot extension. 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by YLSMW** [reply]
@davecheney Hi Dave,
Thanks for reply!
It doesn't work on my side. this solution needs copilot online. But on this issue, it's not yet activated.
![image](https://user-images.githubusercontent.com/101581613/187117574-e3da4a15-0d36-433b-807a-06ac7a078ce1.png)
![image](https://user-images.githubusercontent.com/101581613/187117769-d9d2677a-d170-4797-a46d-bb70e13242e3.png)

**Reply by davecheney** [reply]
@YLSMW 
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by YLSMW** [reply]
@davecheney 
Possible. 
I doubted connection issue as I useWSL2 ubuntu, and the proxy setting is not resolved yet. 
However I am not so sure about this as other extensions passed without error.

![image](https://user-images.githubusercontent.com/101581613/187118132-1318fc5f-b14d-4714-bcb5-20a6c24e1ad3.png)

**Reply by YLSMW** [reply]
And another strange point is that, in the windows base env, copilot works well. But in WSL Ubuntu, it fails.

**Reply by davecheney** [reply]
@YLSMW could you please carefully check that inside your WSL Ubuntu installation you are running VSCode, not the VSCodium or OSSCode source builds. Thank you.

**Reply by YLSMW** [reply]
@davecheney There is perhaps some missunderstanding Sir.
I am running VS code on windows 10. And using WSL2 for a Linux environment.
VScode performs remote-WSL.
The Copilot was OK a week ago but failed to work since last week.

**Reply by davecheney** [reply]
Thank you. I have passed this on to the extension maintainers.

**Reply by YLSMW** [reply]
Thank you so much sir!

**Reply by gesien** [reply]
i have this issue. how you solved?
